### PR TITLE
Added sdcMicro anonymization report

### PR DESCRIPTION
**Pull request by jim-sheldon**

### CODE REVIEW

File: analytics/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
+FROM rocker/r-ver:4.0.3
```
Comment (abhidg) [inline analytics/Dockerfile:1]: Worth using 4.2.1? No idea what the support timescale for R versions is

File: analytics/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+RUN apt-get update \
+	&& apt-get install -y --no-install-recommends \
+  	apt-utils \
+  	ed \
```
Comment (abhidg) [inline analytics/Dockerfile:6]: Something's still using ed?

**Review by abhidg — APPROVED** [review]

File: analytics/Dockerfile (hunk 2 above)
Comment (jim-sheldon) [reply analytics/Dockerfile]: Good catch, I am not sure where that `RUN` block came from, but this image does not need any of those packages.

File: analytics/Dockerfile (hunk 1 above)
Comment (jim-sheldon) [reply analytics/Dockerfile:1]: I get an error building using 4.2.1, will need to investigate more later.

**Review by iamleeg — APPROVED** [review]

**Review by iamleeg — APPROVED** [review]

**Pull request merged**